File: complete/src/main/java/hello/Application.java
package hello;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.bind.RelaxedPropertyResolver;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
@RestController
public class Application {
	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}

	@RequestMapping("/")
	public String home() {
		return "Hello IBM Bluemix!";
	}

	@RequestMapping("/object")
	public SomeObject object() {
		return new SomeObject("some_value");
	}
}

File: complete/src/main/kotlin/org/skinny/drive_to_work/Application.kt
package org.skinny.drive_to_work;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean
import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder
import java.time.LocalDate

@SpringBootApplication
open class Application {
    @Bean
    open fun jacksonBuilder(): Jackson2ObjectMapperBuilder {
        val builder: Jackson2ObjectMapperBuilder = Jackson2ObjectMapperBuilder();
        builder.serializerByType(LocalDate::class.java, LocalDateSerializer());
        return builder;
    }

    @Bean
    open fun episodeRepository(): EpisodeRepository {
        return EpisodeRepository()
    }
}

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args);
}
